v1/metrics.yaml

```diff
@@ -232,6 +232,59 @@ paths:
               uri: '{{options.host}}/pageviews/top/{project}/{access}/{year}/{month}/{day}'
       x-monitor: false
 
+  /unique-devices/{project}/{access-site}/{granularity}/{start}/{end}:
```

Comment:
Hm, but uniques is a completely different metric.  It has nothing to do with pageviews.  We're not putting it higher up because of the ad-block bug, it really is separate.  Hence the desire to rename above.  So, just to be clear, pageviews and unique_devices are coming from two totally different data pipelines that have nothing to do with each other besides the basic varnishkafka -> kafka -> camus that all our webrequest data comes through.